packages/modeling/src/operations/extrusions/extrudeFromSlices.js

```diff
@@ -54,6 +55,9 @@ const extrudeFromSlices = (options, base) => {
 
   if (numberOfSlices < 2) throw new Error('numberOfSlices must be 2 or more')
 
+  // Repair gaps in the base slice
+  repairSlice(base)
```

Comment:
I am personally against automatic repair, but if there is a parameter that controls if repair is called, I will be ok if repair is done by default. I really want to be able to turn off repair as a user.

Reply:
I have updated the code to include an option `repair` which defaults to true, but can be disabled. I felt it was important to default to true, since otherwise many examples would fail to extrude (including the birthday balloons). I hope that over time we can make the BSP tree really solid so that repair is not necessary.